gradle.properties

```diff
@@ -1,6 +1,6 @@
 GROUP=com.squareup.misk
 
-VERSION_NAME=18.9.10-51cc9bc3
+VERSION_NAME=0.5.0-SNAPSHOT
```

Comment:
We're moving back to Snapshots?

Reply:
Nope! It’s just the most convenient thing to put in git since most builds aren’t release builds.

Reply:
Sweet, makes more sense!